src/test/index.spec.js

```diff
@@ -0,0 +1,32 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+describe('Testing home endpoint', () => {
+  it('It should return status of 200 on successful page load', done => {
```

Comment:
Are we loading a page?

Reply:
No we are not loading a page. The test was just to test the functionality of the set up.

Reply:
your test description should clearly explain what's being tested.